In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

In [2]:
datahw01 = pd.read_csv('datasethw01.csv')

In [3]:
datahw01.size, datahw01.shape

(8370, (930, 9))

In [95]:
for i in range(8):
    print(datahw01.iloc[:,i+1].all()==0)


False
True
False
False
True
True
True
False


In [105]:
datahw01['MOS'] = np.rint(datahw01['MOS'])

In [107]:
datahw01.head()

Date Of Test  Signal (dBm)  Speed (m/s)  Distance from site (m)  \
0  2017-07-01 00:00:27           -61    68.800003                 1048.60   
1  2017-07-01 00:02:57           -61    68.769997                 1855.54   
2  2017-07-01 00:05:29           -71    69.169998                 1685.62   
3  2017-07-01 00:08:02           -65    69.279999                 1770.92   
4  2017-07-01 00:10:30          -103     0.820000                  256.07   

   Call Test Duration (s)  Call Test Result  Call Test Technology  \
0                    90.0                 1                     1   
1                    90.0                 1                     1   
2                    90.0                 1                     1   
3                    90.0                 1                     1   
4                    60.0                 1                     1   

   Call Test Setup Time (s)  MOS  
0                      0.56  2.0  
1                      0.45  3.0  
2                      0.51  2.0  
3                      0.00  1.0  
4                      3.35  4.0

In [16]:
datahw01['Call Test Result'].nunique(), datahw01['Call Test Technology'].nunique()

(2, 3)

In [48]:
grouped = datahw01.groupby('Call Test Result')

for name, group in grouped:
    print(group.shape)
    
grouped = datahw01.groupby('Call Test Technology')
print('Tecnologias:')
for name, group in grouped:
    print(name)
    print(group.shape)

(11, 9)
(919, 9)
Tecnologias:
0
(21, 9)
1
(270, 9)
2
(639, 9)



 - Dados: MOS

sucesso no teste = 919  
falhas no teste  = 11

- Tecnologias:

LTE = 639
UMTS = 270
GSM = 21

- 